packages/amplify/amplify_flutter_android/android/build.gradle

```diff
@@ -72,15 +72,15 @@ android {
 dependencies {
     api amplifyCore
 
-    implementation 'com.amplifyframework:core:1.33.0'
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9'
+    implementation 'com.amplifyframework:core:1.36.0-dev-preview.0'
```

Comment:
What is the purpose of using the Android dev-preview?

Reply:
Answered offline. We'll be using native dev-previews for our dev-preview, and likewise for GA.